app/controllers/unwithdraw_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UnwithdrawController < ApplicationController
+  def index
+    @document = Document.find_by_param(params[:id])
```

Comment:
We'll want to use with_current_edition to check that this is actually a usable document.